src/app/modules/verification/__tests__/verification.routes.spec.ts

```diff
@@ -0,0 +1,360 @@
+import bcrypt from 'bcrypt'
+import { getReasonPhrase, StatusCodes } from 'http-status-codes'
+import mongoose from 'mongoose'
+import session, { Session } from 'supertest-session'
+
+import { BasicField, IVerificationSchema } from 'src/types'
+
+import { setupApp } from 'tests/integration/helpers/express-setup'
+import { buildCelebrateError } from 'tests/unit/backend/helpers/celebrate'
+import { generateDefaultField } from 'tests/unit/backend/helpers/generate-form-data'
+import dbHandler from 'tests/unit/backend/helpers/jest-db'
+
+import getVerificationModel from '../verification.model'
+import { VfnRouter } from '../verification.routes'
+
+import {
+  generateFieldParams,
+  MOCK_HASHED_OTP,
+  MOCK_OTP,
+  MOCK_RECIPIENT,
+  MOCK_SIGNED_DATA,
+} from './verification.test.helpers'
+
+const verificationApp = setupApp('/transaction', VfnRouter)
+const VerificationModel = getVerificationModel(mongoose)
+
+jest.mock('nodemailer', () => ({
+  createTransport: jest.fn().mockReturnValue({
+    sendMail: jest.fn().mockResolvedValue(true),
+  }),
+}))
+// Default Twilio export is a function
+jest.mock('twilio', () => () => ({
+  messages: {
+    create: jest.fn().mockResolvedValue({
+      sid: 'mockSid',
+    }),
+  },
+}))
+
+describe('verification.routes', () => {
+  let mockTransaction: IVerificationSchema
+  let mockTransactionId: string
```

Comment:
why use `let` over declaring each one as `const` and initializing in respective blocks? `let` implies that we are mutating the variables, which should not happen if this is a test? (not sure about this) this is additionally complicated by the fact that within each test block, if we modify the variables, the change will be visible throughout the `describe` block, which might complicate testing later on